ocfweb/docs/docs/staff/backend/prometheus.md

```diff
@@ -0,0 +1,44 @@
+[[!meta title="Prometheus"]]
+
+We use Prometheus to provide real-time monitoring of our hardware. The master is [[dementors|doc staff/backend/servers]] which
+uses the Node Exporter to collect data from other servers.
+
+We monitor servers, desktops, and staff VMs, but not the hozer boxes.
+Additionally, we don't receive email alerts for staff VMs. Monitoring for the networking switch, blackhole, is currently under development.
+
+## Alerts
+
+Alerts can be viewed at [prometheus.ocf.berkeley.edu/alerts](https://prometheus.ocf.berkeley.edu/alerts). They are configured at [this folder][prometheus-puppet] in the Puppet configs.
+
+Alerts are currently under development and may not be fully comprehensive.
+
+## Custom Queries
+
+Prometheus supports querying a wide variety of metrics. (For a full list, go to [Prometheus](https://prometheus.ocf.berkeley.edu) and use the "insert metric at cursor" dropdown.) A basic query comes in the form:
```

Comment:
I'd avoid documenting non-OCF specifics of Prometheus, since their documentation will always be better. Examples that actually use OCF queries are fine, though.

Reply:
It'd be nice if we could link to said documentation here (or refer to it).